### ISSUE TITLE
Help PyPA with Warehouse (next gen PyPI) testing

### ISSUE THREAD

**Issue by aclark4life** [python-pillow/Pillow#3010]
Hi @python-pillow/pillow-team ,

Just got an email from @pypa re: testing Warehouse (next gen PyPI via @dstufft), which I think we should try to do if possible. Can we tentatively plan to upload the next release to both PyPI and Warehouse?

More here:

- https://wiki.python.org/psf/WarehousePackageMaintainerTesting

**Comment by wiredfool** [comment]
Yep, got that as well. I've been meaning to get a .0.1 release out, but I've not had time this month.

**Comment by hugovk** [comment]
The new PyPI (Warehouse) was launched yesterday.

https://pypi.org/

https://wiki.python.org/psf/WarehouseRoadmap
